esrally/track/params.py

```diff
@@ -580,6 +580,21 @@ def percent_completed(self):
         return self.current_bulk / self.total_bulks
 
 
+class ForceMergeParamSouce:
+    def __init__(self, track, params, **kwargs):
```

Comment:
Can you please add tests in `tests/track/params_test.py`? You can have a look at `DeleteIndexParamSourceTests` for inspiration. IMHO it should check all three cases how the  `index` parameter can be set (via the track, fallback via `_all` and explicitly set via the `index` parameter on the track).

Reply:
Thanks for the pointer, added tests to cover the three cases as you mentioned.

Reply:
`make precommit` is failing now with:

```

08:13:11 + make precommit
08:13:19 ************* Module esrally.track.params
08:13:19 esrally/track/params.py:584:4: W0231: __init__ method from base class 'ParamSource' is not called (super-init-not-called)
```

can you please add the following in the first line of the constructor:

```python
super().__init__(track, params, **kwargs)
```